### PR TITLE
IW-1489 | add rail logic + backend urls

### PR DESCRIPTION
**Pull request by xkxd**
https://wikia-inc.atlassian.net/browse/IW-1489

### CODE REVIEW

File: extensions/wikia/Recirculation/RecirculationHooks.class.php [hunk 1]
```diff
@@ -15,8 +15,11 @@ class RecirculationHooks {
 	 * @return bool
 	 */
 	public static function onBeforePageDisplay( OutputPage $out, Skin $skin ) {
+		global $wgDevelEnvironment;
+
 		JSMessages::enqueuePackage( 'Recirculation', JSMessages::EXTERNAL );
 		Wikia::addAssetsToOutput( 'recirculation_scss' );
+		$out->addJsConfigVars('wgServiceUrl', empty( $wgDevelEnvironment ) ? 'services.wikia.com' : 'services.wikia-dev.pl' );
```
Comment (mszabo-wikia) [inline extensions/wikia/Recirculation/RecirculationHooks.class.php]: `wgServicesExternalDomain` is already available on the client side and should be used instead

**Review by mszabo-wikia — CHANGES_REQUESTED** [review]

File: extensions/wikia/Recirculation/js/helpers/sponsoredContent.js [hunk 2]
```diff
@@ -1,59 +1,30 @@
...
-				"title": "Kermit",
-				"attribution": "Netflix"
-			}
-		];
+		if (!hasFetched) {
+			hasFetched = true;
 
-		return mock;
-		// return $.ajax();
+			$
```
Comment (jsutterfield) [inline extensions/wikia/Recirculation/js/helpers/sponsoredContent.js]: This formatting is kind of funky. Maybe bring `.ajax` up to this line?

File: extensions/wikia/Recirculation/js/helpers/sponsoredContent.js [hunk 3]
```diff
@@ -1,59 +1,30 @@
...
+				.ajax({
+					url: w.wgServicesExternalDomain + 'wiki-recommendations/sponsored-articles',
```
Comment (jsutterfield) [inline extensions/wikia/Recirculation/js/helpers/sponsoredContent.js]: String interpolation via backticks is typically preferred, yea?

**Review by jsutterfield — APPROVED** [review]

File: extensions/wikia/Recirculation/js/helpers/sponsoredContent.js (hunk 3 above)
Comment (mszabo-wikia) [reply extensions/wikia/Recirculation/js/helpers/sponsoredContent.js]: It won't work in IE11

**Review by mszabo-wikia — APPROVED** [review]

**Pull request merged**